charts/csi-cloudscale/templates/storageclass.yaml

```diff
@@ -0,0 +1,24 @@
+{{- $csi := .Values.csi }}
+{{- $provisioner := .Values.provisioner }}
+{{- range $storageClass := .Values.csi.storageClasses }}
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: {{ $storageClass.name }}
+{{- if $storageClass.default }}
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+{{- end }}
+provisioner: {{ $storageClass.provisioner | default $provisioner.name }}
+allowVolumeExpansion: {{ $storageClass.allowVolumeExpansion | default $csi.allowVolumeExpansion }}
```

Comment:
We are currently using the attributes `reclaimPolicy` and `volumeBindingMode`. Not sure if the csi driver can hanlde [mountOptions](https://kubernetes.io/docs/concepts/storage/storage-classes/#mount-options).

Reply:
mountOptions are not supported. Feel free to raise an issue or drop me a message, if you require them.

For `reclaimPolicy` and `volumeBindingMode` I can create values that you can override. Would that work for you?

Reply:
Perfect!